pages/404.tsx

```diff
@@ -25,7 +25,7 @@ export default function ErrorPage() {
           </header>
         </div>
         <div>
-          <img src={image} className='mx-auto w-52' />
+          <img src={image} className='mx-auto w-52' alt='404' />
```

Comment:
You can skip the alt atttribute in images for now. It will only show warning inside eslint.